### DISCUSSION TITLE
Augmentation of Transfer Process Protocol (for Consent Management of Personal Data)

### DISCUSSION THREAD

**Discussion by anilturkmayali** [eclipse-dataspace-protocol-base/DataspaceProtocol#31] [Ideas]
Hi all, 
There is this idea rom Felix Bole (Visions) to contribute to the extension of the protocol on the end of consent management: 
https://gist.github.com/FelixBole/f857176bea92f8c5bebb47b70cae39da

For your consideration.

**Comment by arnoweiss** [comment]
Consent is handled by underlying authentication protocols like DCP (which has a [chapter in the spec](https://eclipse-dataspace-dcp.github.io/decentralized-claims-protocol/v1.0-RC4/#consent) devoted to consent). I think this is quite far out of scope of the DSP spec.
